scripts/git-utils/checkout-pr-if-exist.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# get all opened PRs  with specified label
+RESULT=`curl -s https://api.github.com/search/issues?q=is:pr%20state:open%20label:$1%20repo:OlegZharkov/content`
```

Comment:
@OlegZharkov I'm not sure I completely understand the filter on the API here but `repo:OlegZharkov/content` looks wrong.

Reply:
@hmenager you're right. Sorry, I was using my fork for debugging and forgot and change it back